.skiff/kube.yaml

```diff
@@ -1,99 +0,0 @@
-apiVersion: extensions/v1beta1
-kind: Ingress
-metadata:
-  name: lm-explorer-ingress
-  namespace: lm-explorer
-  annotations:
-    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
-    kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/ssl-redirect: "true"
-    nginx.ingress.kubernetes.io/enable-cors: "false"
-spec:
-  tls:
-    - secretName: lm-explorer-tls
-      hosts:
-      - lm-explorer.apps.allenai.org
-      - gpt2.apps.allenai.org
-  rules:
-  - host: lm-explorer.apps.allenai.org
-    http:
-      paths:
-      - backend:
-          serviceName: lm-explorer
-          servicePort: 80
-  - host: gpt2.apps.allenai.org
-    http:
-      paths:
-      - backend:
-          serviceName: lm-explorer
-          servicePort: 80
----
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  labels:
-    app: lm-explorer
-  name: lm-explorer
-  namespace: lm-explorer
-spec:
-  replicas: 1
-  strategy:
-    rollingUpdate:
```

Comment:
@codeviking is this something I should preserve?

Reply:
Yes, I think so.

I believe so.  It should prevent downtime, as it's effectively a blue / green deploy.